File: 240102/가장 짧은 문자열/shortest-string.java
import java.util.Scanner;
public class Main {
    public static void main(String[] args) {
        // 여기에 코드를 작성해주세요.
        Scanner sc = new Scanner(System.in);
        int max_val = 0 ;
        int min_val = 21;

        for(int i= 0 ; i<3; i++){
            String word = sc.next();
            if (word.length()>max_val){
                max_val = word.length();
            }
            if (word.length()<min_val){
                min_val = word.length();
            }
        }
        System.out.print(max_val- min_val);
    }
}

File: 240102/가장 짧은 문자열/shortest-string.py
max_val = 0 
min_val = 21
for _ in range(3):
    name = input()
    if len(name)>max_val:
        max_val = len(name)
    
    if len(name) < min_val:
        min_val = len(name)
print(max_val - min_val)